datanode/sqlstore/ledgerentry_filter.go

```diff
@@ -180,3 +137,28 @@ func prepareGroupFields(groupByAccountField []entities.AccountField, groupByLedg
 
 	return fields
 }
+
+var ErrLedgerEntryFilterForParty = fmt.Errorf("filtering ledger entries should be limited to a single party")
```

Comment:
We usually add those at the tops of files. also seing there's not formatting directiver, errors.New() should suffice.

Reply:
Addressed here https://github.com/vegaprotocol/vega/pull/6557/commits/92438786697c4248d2d0c9c989d79eaf938c7778#diff-149a70d32b3e4f50b3d653208267b371f6335459774f97eb3dd122023e3432fcR11-R12
Thanks!